examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+    "        x = F.relu(self.fc2(x))\n",
+    "        x = self.fc3(x)\n",
+    "        return x\n",
+    "\n",
+    "\n",
+    "net = Net()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "cuda:0\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "Net(\n",
+       "  (conv1): Conv2d(3, 6, kernel_size=(5, 5), stride=(1, 1))\n",
+       "  (pool): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)\n",
+       "  (conv2): Conv2d(6, 16, kernel_size=(5, 5), stride=(1, 1))\n",
+       "  (fc1): Linear(in_features=400, out_features=120, bias=True)\n",
+       "  (fc2): Linear(in_features=120, out_features=84, bias=True)\n",
+       "  (fc3): Linear(in_features=84, out_features=10, bias=True)\n",
+       ")"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Assume that we are on a CUDA machine, then this should print a CUDA device:\n",
+    "device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",
+    "print(device)\n",
+    "\n",
+    "net.to(device)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's setup an optimizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch.optim as optim\n",
+    "\n",
+    "criterion = nn.CrossEntropyLoss()\n",
+    "optimizer = optim.SGD(net.parameters(), lr=0.001, momentum=0.9)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's setup a simple training loop"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 1/2, Iteration 2000/12500: Loss = 2.1585227974653245\n",
+      "Epoch 1/2, Iteration 4000/12500: Loss = 1.8301588917076588\n",
+      "Epoch 1/2, Iteration 6000/12500: Loss = 1.673056129336357\n",
+      "Epoch 1/2, Iteration 8000/12500: Loss = 1.5852073608338832\n",
+      "Epoch 1/2, Iteration 10000/12500: Loss = 1.5238751489222049\n",
+      "Epoch 1/2, Iteration 12000/12500: Loss = 1.4723967086076737\n",
+      "Epoch 2/2, Iteration 2000/12500: Loss = 1.403484536588192\n",
+      "Epoch 2/2, Iteration 4000/12500: Loss = 1.3825845147520304\n",
+      "Epoch 2/2, Iteration 6000/12500: Loss = 1.343847774296999\n",
+      "Epoch 2/2, Iteration 8000/12500: Loss = 1.3481838842481375\n",
+      "Epoch 2/2, Iteration 10000/12500: Loss = 1.3130395187735557\n",
+      "Epoch 2/2, Iteration 12000/12500: Loss = 1.2754147632271051\n",
+      "Finished Training in 117.91511368751526 seconds.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from time import time\n",
+    "\n",
+    "no_epochs = 2\n",
```

Comment:
Why call it "**no_epochs**"? I think maybe "epochs" or "num_epochs" is more clear.

Reply:
Original tutorial called it no_epochs. Also no_ is common alternative of num_

Reply:
I didn't find `no_epoch` in the original tutorial, just:
```py
for epoch in range(2):
    ... ...
```